### PR TITLE
Inline `falafel`.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
See [comment](https://github.com/royriojas/esformatter-jsx/issues/31#issuecomment-261300358).
It's very rough, but how does it look?  Would it be considered?

### CODE REVIEW

**Comment by royriojas** [comment]
hi @wtgtybhertgeghgtwtg thanks for your contribution. I would like understand better why this is needed. While I do like the idea of inlining falafel I would like to consider also changing the node version to at least node 4. As shown in http://node.green/ that version won't need isArray nor Object.keys. Just make sure tests are passing and will merge in order to remove deps.

**Comment by wtgtybhertgeghgtwtg** [comment]
It is not really a need, but it would make things simpler.  You wouldn't need to maintain `fresh-falafel`, for one, and you wouldn't have to worry about `acorn` causing issues with CSP, since it flat out drops the `acorn` dependency.
I have been having trouble with `esbeautifier` beautifier on my machine, but I will continue to try and resolve that so that I can get the tests passing.

**Comment by royriojas** [comment]
Thanks @wtgtybhertgeghgtwtg, again thanks for your PR. Looking forward to your changes.

**Comment by wtgtybhertgeghgtwtg** [comment]
It looks like `esbeautifier` needs a `.esformetter` file to work, but I cannot seem to find one for this repository.  Am I missing something?

**Comment by wtgtybhertgeghgtwtg** [comment]
Tests pass.  Using single quotes in `scripts` is causing issues on Windows, though.

**Comment by royriojas** [comment]
@wtgtybhertgeghgtwtg can you please elaborate? what kind of issues?

**Comment by wtgtybhertgeghgtwtg** [comment]
For example, [the `beautify-check` script](https://github.com/royriojas/esformatter-jsx/blob/master/package.json#L27) will end up passing
`["{DIRECTORY}/'index.js'", "{DIRECTORY}/'lib/**/*.js'", "{DIRECTORY}/'specs/**/*.spec.js'"]`
instead of
`["{DIRECTORY}/index.js", "{DIRECTORY}/'lib/**/*.js", "{DIRECTORY}/specs/**/*.spec.js"]`
to `glob-expand`.

**Comment by royriojas** [comment]
@wtgtybhertgeghgtwtg are u sure? I tested this in windows 7 (a VM) and it seems it is working as expected.

**Comment by royriojas** [comment]
In any case let me investigate about it.

On the other hand... can we get rid of object-keys as a dependency? we need to bump the travis.yml to node 4, and 6, I guess by now is safe to ignore node < 1

**Comment by wtgtybhertgeghgtwtg** [comment]
`object-keys` removed.

**Comment by royriojas** [comment]
Thank you very much @wtgtybhertgeghgtwtg

File: lib/format-jsx.js [hunk 1]
```diff
@@ -332,7 +332,8 @@ module.exports = {
                 line = line.split( key ).join( parts.join( '\n' ) );
                 line = line.replace( /\s+$/, '' );
 
-                delete expressionContainers[ key ];
+                delete expressionContainers[ key ]
```
Comment (royriojas) [inline lib/format-jsx.js]: oh crap, this is a bug on esformatter... forget to mention how to fix this... will fix in master before publishing a new one.

**Pull request merged**